var/spack/repos/builtin/packages/resolve/package.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Resolve(CMakePackage, CudaPackage, ROCmPackage):
+    """ReSolve is a library of GPU-resident sparse linear solvers. It contains iterative and direct
+    solvers designed to run on NVIDIA and AMD GPUs, as well as CPU devices."""
+
+    homepage = "https://github.com/ORNL/ReSolve"
+    url = "https://github.com/ORNL/ReSolve.git"
+    maintainers("cameronrutherford", "pelesh", "ryandanehy", "kswirydo")
+
+    # version("1.0.0", submodules=False, branch=develop)
+    # version("develop", submodules=False, branch=develop)
+
+    variant("klu", default=True, description="Use KLU, AMD and COLAMD Libraries from SuiteSparse")
+
+    depends_on("suite-sparse", when="+klu")
+
+
+    with when("+rocm"):
+      # Need at least 5.6+
+      depends_on("rocsparse@5.6:")
+      depends_on("rocblas@5.6:")
+      depends_on("rocsolver@5.6:")
+
+      # Optional profiling dependecies
+      # Will be controlled by variant in the future
+      depends_on("roctracer-dev@5.6:")
+      depends_on("roctracer-dev-api@5.6:")
+      depends_on("rocprofiler-dev@5.6:")
```

Comment:
```suggestion
      # depends_on("roctracer-dev@5.6:")
      # depends_on("roctracer-dev-api@5.6:")
      # depends_on("rocprofiler-dev@5.6:")
```

These are broken for now, but will be included when issues are resolved.